tests/core/document/test_partial_document_update.py

```diff
@@ -133,7 +133,7 @@ def _set_up_for_text_field_test(self):
             "text_field_tensor": "text field tensor",
             "_id": "1"
         }
-        tensor_search.add_documents(config=self.config, add_docs_params=AddDocsParams(
+        self.add_documents(config=self.config, add_docs_params=AddDocsParams(
```

Comment:
I have extracted an add_documents method in the base class in marqo_test.py.  It will force refresh the index after adding documents. This is to guarantee the following search request can get the latest index settings so the integration tests can pass. 
In the following changes, I'll delete the add_documents method from tensor_search module, and use the one I moved to core/document module

Reply:
rename the method to reflect the intention to refresh cache